src/core/internal/dassert.d

```diff
@@ -80,23 +85,48 @@ string _d_assert_fail(A, B)(const scope string comp, auto ref const scope A a, a
     string valA = miniFormatFakeAttributes(a);
     string valB = miniFormatFakeAttributes(b);
     immutable token = invertCompToken(comp);
-    return combine(valA, token, valB);
+    return combine([valA], token, [valB]);
```

Comment:
```suggestion
    return combine(&valA[0 .. 1], token, &valB[0 .. 1]);
```

Reply:
Why? 
The current form is already `@nogc` due to the `scope` parameter.

Reply:
Ah, nvm then